src/Markdig/Parsers/NumberedListItemParser.cs

```diff
@@ -40,6 +40,7 @@ public override bool TryParse(BlockProcessor state, char pendingBulletType, out
                 {
                     startChar = endChar;
                 }
+                state.NextChar();
```

Comment:
What is this skipping? Add a comment if necessary

Reply:
placed on todo